### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by hafarfur**
First edit of my readme

### CODE REVIEW

File: student-work/2023/Max-Zingerman/readme.md [hunk 1]
```diff
@@ -1 +1,4 @@
-
+# Max's readme file
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Max-Zingerman/readme.md:1]:
```suggestion
# Max's Readme File 
```

File: student-work/2023/Max-Zingerman/readme.md [hunk 2]
```diff
@@ -1 +1,4 @@
-
+# Max's readme file 
+This is going to be my readme file
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Max-Zingerman/readme.md:2]:
```suggestion
This is going to be my readme file.  
```

File: student-work/2023/Max-Zingerman/readme.md [hunk 3]
```diff
@@ -1 +1,4 @@
-
+# Max's readme file 
+This is going to be my readme file  
+I don't have a portfolio yet nor a LinkedIn account
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Max-Zingerman/readme.md:3]:
```suggestion
I don't have a portfolio yet, nor do I have a LinkedIn account.  
```

File: student-work/2023/Max-Zingerman/readme.md [hunk 4]
```diff
@@ -1 +1,4 @@
-
+# Max's readme file 
+This is going to be my readme file  
+I don't have a portfolio yet nor a LinkedIn account  
+Can't think of something to add here now. I'll add things here in the future.
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Max-Zingerman/readme.md:4]:
```suggestion
I can't think of something to add here now. I'll add things here in the future.   
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
Please add your CV, it doesn't have to be the same as your LinkedIn profile. 
If you do not have one at all this is a bigger issue.

**Comment by hafarfur** [comment]
PR

**Review by Laura-Novich-OBW — APPROVED** [review]

**Comment by Laura-Novich-OBW** [comment]
much better!

**Pull request merged**